include/oboe/FullDuplexStream.h

```diff
@@ -24,26 +24,68 @@
 
 namespace oboe {
 
-class FullDuplexStream : public AudioStreamCallback {
+/**
+ * FullDuplexStream can be used to synchronize an input and output stream.
+ *
+ * For the builder of the output stream, call setDataCallback() with this stream.
+ *
+ * When both streams are ready, onAudioReady() will call onBothStreamsReady().
+ * Callers must override onBothStreamsReady().
+ *
+ * To ensure best results, open an output stream before the input stream.
+ * Call inputBuilder.setBufferCapacityInFrames(mOutputStream->getBufferSizeInFrames() * 2).
```

Comment:
Also make sure the input stream uses the same sample rate as the output stream.
Some devices have different defaults for input and output sample rate!

    inputBuilder.setSampleRate(mOutputStream->getSampleRate());